src/main/java/amazin/service/AmazonService.java

```diff
@@ -0,0 +1,83 @@
+package amazin.service;
+
+import com.amazonaws.auth.AWSCredentials;
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.s3.model.CannedAccessControlList;
+import com.amazonaws.services.s3.model.DeleteObjectRequest;
+import com.amazonaws.services.s3.model.PutObjectRequest;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+
+import javax.annotation.PostConstruct;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.Date;
+
+@Service
+public class AmazonService {
```

Comment:
maybe this should be named PictureService?

Reply:
or maybe FileService if we will later support other file storage too, just considering this because looking at the name of the file it seems weird to have AmazonService within an app named Amazin-online-bookstore so we could abstract a bit away from the implementation choice of Amazon AWS in the file name

Reply:
For our purpose, we should only allow pictures to be stored, therefore PictureService is good enough.